samples/unit-tests/series-bubble/marker/demo.js

```diff
@@ -108,4 +108,75 @@ QUnit.test('Bubble data points without z-param.(#8608)', function (assert) {
         'object',
         'Has marker'
     );
+});
+
+QUnit.test('Bubble animation and async redraws (#13494)', assert => {
+    const clock = TestUtilities.lolexInstall();
+
+    try {
+
+        const chart = Highcharts.chart('container', {
+            chart: {
+                type: 'bubble'
+            },
+            plotOptions: {
+                series: {
+                    animation: {
+                        duration: 100
+                    }
+                }
+            }
+        });
+
+        chart.addSeries({
+            data: [
+                [9, 81, 10],
+                [3, 52, 9],
+                [31, 18, 47],
+                [79, 91, 13],
+                [93, 23, -27],
+                [44, 83, -28]
+            ]
+        });
+
+        assert.strictEqual(
+            chart.series[0].points[0].graphic.attr('width'),
+            1,
+            'Points should be in animation start position'
+        );
+        setTimeout(() => {
+            chart.addSeries({
+                data: [
+                    [13, 30, 10],
+                    [23, 20, -10],
+                    [23, 40, 10]
+                ]
+            });
+            assert.notEqual(
+                chart.series[0].points[0].graphic.attr('width'),
+                1,
+                'First series points should continue animating'
+            );
+            assert.strictEqual(
+                chart.series[1].points[0].graphic.attr('width'),
+                1,
+                'Second series points should be in animation start position'
+            );
+        }, 50);
+
+        setTimeout(() => {
```

Comment:
Since it's async test, shouldn't we use `done = asert.async()`? See: https://api.qunitjs.com/assert/async

Reply:
Actually that's not required when we use lolex, since `setTimeout()` is overridden and made sync.